docs/content/en/docs/installation/runtime-hooks.md

```diff
@@ -76,3 +76,100 @@ Output:
 ```json
 {"time":"2024-07-01T10:57:21.435689144Z","level":"INFO","msg":"hook request to agent succeeded","hook":"create-container","start-time":"2024-07-01T10:57:21.433755984Z","req-cgroups":"/kubepods/besteffort/podd4e74de2-0db8-4143-ae55-695b2489c727/crio-828977b42e3149b502b31708778d0c057efbce038af80d0882ed3e0cb0ff8796","req-rootdir":"/run/containers/storage/overlay-containers/828977b42e3149b502b31708778d0c057efbce038af80d0882ed3e0cb0ff8796/userdata","req-containerName":"test"}
 ```
+
+## Containerd
+
+### Setup Minikube
+
+```shell
+minikube start --driver=kvm2 --container-runtime=containerd
+minikube image load cilium/tetragon:latest
+minikube image load cilium/tetragon-operator:latest
+minikube image load cilium/tetragon-rthooks:latest
+minikube image ls | grep tetragon
+```
+
+Output should be similar to:
+```
+docker.io/cilium/tetragon:latest
+docker.io/cilium/tetragon-rthooks:latest
+docker.io/cilium/tetragon-operator:latest
+```
+
+Tetragon Runtime Hooks use [NRI](https://github.com/containerd/nri). NRI is [enabled by
+default](https://github.com/containerd/containerd/blob/main/docs/NRI.md#disabling-nri-support-in-containerd)
+starting from containerd version 2.0. For version 1.7, however, it needs to be enabled in the
+configuration.
+
+```shell
+minikube ssh cat /etc/containerd/config.toml > /tmp/old-config.toml
+./contrib/tetragon-rthooks/tetragon-oci-hook-setup patch-containerd-conf enable-nri --config-file=/tmp/old-config.toml --output=/tmp/new-config.toml
+diff -u /tmp/old-config.toml /tmp/new-config.toml
+```
+
+Output should be something like:
+
+```diff
+--- /tmp/old-config.toml        2024-07-02 11:51:23.893382357 +0200
++++ /tmp/new-config.toml        2024-07-02 11:51:52.841533035 +0200
+@@ -67,3 +67,11 @@
+     mutation_threshold = 100
+     schedule_delay = "0s"
+     startup_delay = "100ms"
++  [plugins."io.containerd.nri.v1.nri"]
++    disable = false
++    disable_connections = false
++    plugin_config_path = "/etc/nri/conf.d"
++    plugin_path = "/opt/nri/plugins"
++    plugin_registration_timeout = "5s"
++    plugin_request_timeout = "2s"
++    socket_path = "/var/run/nri/nri.sock"
+```
+
+Install the new configuration file and restart containerd
+```shell
+cp /tmp/new-config.toml /etc/containerd/config.toml
```

Comment:
Shouldn't we copy that inside minikube again? These commands seems to change the host `/etc/containerd/config.toml` file (and not the one inside minikube VM).

Reply:
Nice catch! Yap, there is a `minikube` missing in front of the `cp`.